src/autoplex/data/phonons/flows.py

```diff
@@ -393,7 +393,7 @@ def make(
             supercell_matrix_job = reduce_supercell_size_job(
                 structure=structure,
                 min_length=self.supercell_settings.get("min_length", 12),
-                max_length=self.supercell_settings.get("max_length", 25),
+                max_length=self.supercell_settings.get("max_length", 20),
```

Comment:
@JaGeo this was why I had so large rattled supercells

Reply:
But why not phonon ones?

Reply:
There we set no max_length and I think it takes the fallback value in atomate2? I can't check the atomate2 code now.

Reply:
I am confused. I thought we would use matrices for the supercells in both cases. At least, that is what i wanted to implement

Reply:
You mean the same matrices? But it was implemented as separate ones. I can make it that it's the same matrix. Maybe in another PR?

Reply:
There should be an option to make them the same.